core/src/main/java/com/linecorp/armeria/common/stream/StreamWriter.java

```diff
@@ -33,6 +35,32 @@
      */
     boolean isOpen();
 
+    /**
+     * Writes the specified object to the {@link StreamMessage}. The written object will be transferred to the
+     * {@link Subscriber}.
+     *
+     * @throws IllegalArgumentException if the stream was already closed or the publication of the specified
+     *         object has been rejected
```

Comment:
`IllegalStateException` is raised when stream was closed.

Reply:
Urgh - too much coffee can't read

Reply:
covfefe